### PR TITLE
Update RxAndroid Version.And gradle plugin version.

### PR DESCRIPTION
**Pull request by wind0ws**
Also add  proguard-rules for RxJava,It is useful if developer enable minify in theirs app with RxCamera library.Otherwise app will crash in some case.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+#
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+#Tue Mar 22 08:44:20 CST 2016
+systemProp.http.proxyHost=127.0.0.1
+systemProp.http.proxyPort=8087
```
Comment (sibelius) [inline gradle.properties]: could you remove this?

Comment (wind0ws) [inline gradle.properties]: of course. Sorry for that: I can't connect to google without internet proxy.

**Comment by sibelius** [comment]
could you please rebase to merge your commits into only one?

File: gradle.properties [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
-
+#
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+#Tue Mar 22 08:44:20 CST 2016
+#systemProp.http.proxyHost=127.0.0.1
```
Comment (ragnraok) [inline gradle.properties]: can you remove this proxy setting?

**Comment by sibelius** [comment]
@wind0ws I've merged manually with this commit https://github.com/ragnraok/RxCamera/commit/b98451acbb8dde9cf9bf97d2d21088dba5978982, thanks

**Pull request closed without merging**